R/pkg/R/DataFrame.R

```diff
@@ -2121,7 +2121,7 @@ setMethod("arrange",
           })
 
 #' @rdname arrange
-#' @name orderBy
```

Comment:
Hmm... The generated HTMLs look almost the same (except for some punctuations) as the ones above.